lib/jekyll-gist/gist_tag.rb

```diff
@@ -41,15 +41,15 @@ def render(context)
       private
 
       def determine_arguments(input)
-        matched = input.match(/\A([\S]+|.*(?=\/).+)\s?(\S*)\Z/)
+        matched = input.match(%r!\A([\S]+|.*(?=\/).+)\s?(\S*)\Z!)
         [matched[1].strip, matched[2].strip] if matched && matched.length >= 3
       end
 
       private
 
       def context_contains_key?(context, key)
         if context.respond_to?(:has_key?)
-          context.has_key?(key)
```

Comment:
Nope. Look at the context; this needs to be `has_key`

Reply:
Thanks @pathawks, I kinda suspected this, but `script/cibuild`didn't fail locally.